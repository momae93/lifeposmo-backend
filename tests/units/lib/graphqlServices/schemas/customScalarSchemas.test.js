const customScalarSchemas = require('@GraphqlService/schemas/customScalarSchemas');

describe('[LIB][GRAPHQL][RESOLVERS] : customScalarSchemas', () => {
  it('should match snapshot lib graphql customScalarSchemas', async () => {
    expect(customScalarSchemas).toMatchSnapshot();
  });
});
