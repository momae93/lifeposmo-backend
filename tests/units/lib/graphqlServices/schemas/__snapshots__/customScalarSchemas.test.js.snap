// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[LIB][GRAPHQL][RESOLVERS] : customScalarSchemas should match snapshot lib graphql customScalarSchemas 1`] = `
Object {
  "definitions": Array [
    Object {
      "description": undefined,
      "directives": Array [],
      "kind": "ScalarTypeDefinition",
      "name": Object {
        "kind": "Name",
        "value": "Date",
      },
    },
  ],
  "kind": "Document",
  "loc": Object {
    "end": 17,
    "start": 0,
  },
}
`;
